viewer/vueapp/tests/consts.js

```diff
@@ -519,4 +519,6 @@ export const fields = [{
 export const fieldsMap = {};
 for (const field of fields) {
   fieldsMap[field.exp] = field;
+  fieldsMap[field.dbField] = field;
+  fieldsMap[field.dbField2] = field;
```

Comment:
dbField2 can be undefined, might not matter for tests

Reply:
I always define dbField2 in the tests, so this should be fine